service.autostop/service.py

```diff
@@ -54,66 +59,48 @@ def onPlayBackStopped(self):
 while not monitor.abortRequested():
 
     pacount += 1
-    if pacount % 10 == 0:                          # Check for paused video every 10 seconds
+    if pacount % 10 == 0:                            # Check for paused video every 10 seconds
         try:
             pastoptime = int(settings('pastop'))
             xbmc.log('Autostop pause count and stop time ' + str(pacount) + ' ' + str(pastoptime) +    \
             ' ' + str(player.paflag), xbmc.LOGDEBUG)
             if pastoptime > 0 and pacount >= pastoptime * 60 and player.paflag == 1:
-                if xbmc.Player().isPlayingVideo():
-                    ptag = xbmc.Player().getVideoInfoTag()
-                    ptitle = ptag.getTitle()
-                elif xbmc.Player().isPlayingAudio():
-                    ptag = xbmc.Player().getMusicInfoTag()
-                    ptitle = ptag.getTitle()                
-                else:
-                    ptitle = "playing file"
                 pos = xbmc.Player().getTime()
-                xbmc.Player().stop()
+                logmsg = 'Autostop stopped paused playback: '
+                notifymsg = 'Autostop Paused Timer'
```

Comment:
If on the GUI the strings needs to be localized

Reply:
This has been updated to localized strings.